lib/config/config.go

```diff
@@ -65,6 +65,8 @@ var (
 	DefaultDiscoveryServers = append(DefaultDiscoveryServersV4, DefaultDiscoveryServersV6...)
 	// DefaultTheme is the default and fallback theme for the web UI.
 	DefaultTheme = "default"
+	// DefaultSentyDSN is the default DSN used for sentry reporting
+	DefaultSentryDSN = "http://badeb4c1239b4b7281b2a37a7ca17560@sentry.syncthing.net:9000/3"
```

Comment:
We should have this be just `https://sentry.syncthing.net/push` or something like that, and add the magic ID in the Caddy proxying. We might need to rotate the ID/key (it seems like you can't *set* this explicitly in the Sentry UI, and we might change the server, for example) and we don't want the constant compiled everywhere then...

Reply:
You can't do that. The client does explicit parsing of the user password stuff and uses it for header generation

Reply:
Or I guess we could embed some garbage in the user password and set it at the reverse proxy. Perhaps that would work.

Reply:
Mmm we need to do *something* anyway if we use this, as we can't really deploy it tied to a key that we can't easily regenerate or move to another machine etc